express-post-json/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+let grade = {};
+let nextId = 1;
+const gradesArray = [];
```

Comment:
All new grades should be store in the object on line 3. The array should be generated on-demand in the GET route. Each new grade that is POSTed should be assigned to a key of the object.